engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -3017,6 +3014,7 @@ public static StreamingContainerManager getInstance(RecoveryHandler rh, LogicalP
 
         // restore checkpoint info
         plan.syncCheckpoints(scm.vars.windowStartMillis, scm.clock.getTime());
+        plan.setContextAfterDAGRecovery();
```

Comment:
with additional parameter it is not needed. But let me know which is the preferred way?